docs/zh-cn/translation-guide.md

```diff
@@ -0,0 +1,140 @@
+# 针对 MDN 文档的本地化指南
+
+本文是针对简体中文（zh-CN）文档的翻译指南。如果你所使用的语言尚未有翻译指南，而你又希望新建的话，欢迎你联系[我们负责该语言的翻译团队](https://github.com/mdn/translated-content/blob/docs-readme/README.md)。
+
+同样地，如果你对本文的指南有任何改进的建议，请提出问题（issue）或发起 PR。我们对此表示欢迎。下面进入简体中文翻译指南的正文部分。
+
+> MDN 中的文档正在逐渐向 Markdown 格式迁移（由 [yari 平台渲染](https://github.com/mdn/yari) 为 HTML），以下的说明仅针对 Markdown 格式，对于现有的 HTML 格式文档，并非完全适用。若你对 MDN 上使用的 Markdown 格式不熟悉，请参阅 [MDN 的 Markdown](https://developer.mozilla.org/zh-CN/docs/MDN/Contribute/Markdown_in_MDN)。
+
+## 翻译标题和 ID
+
+文档中的目录导航、与代码块相关联的运行实例等会使用到 HTML ID。而对于 Markdown 文档，其渲染得到的 HTML 文件的 ID 则与 Markdown 文档中的标题一一对应（这并不意味着两者相等，yari 平台对于半角空格、标点等有特殊处理）。
+
+例如，对于标题：
+
+```md
+## Examples
+```
+
+在 `zh-CN` 中的翻译应该是：
+
+```md
+## 示例
+```
+
+## 翻译示例代码块
+
+对于英文的示例代码块，可以选择翻译其中的注释、字符串、输出结果等等，但不建议对变量、函数、方法等名称进行翻译。
+
+一些示例代码块会有与之对应的“实例”（即对示例代码运行效果的实时展示）。这些实例是通过 `{{EmbedLiveSample}}` 等宏实现的，因为在本地化文档中会翻译英文标题，请**注意一并修改运行实例宏的第一个参数为相对应的中文标题**（参见：[运行实例](https://developer.mozilla.org/zh-CN/docs/MDN/Structures/Live_samples)）。
+
+## 站内链接翻译
+
+在翻译文档时，请记得将 MDN 站内网页的链接中的语言替换为 `zh-CN`，以便读者能够访问与之对应的本地化文档。
+
+例如：
+
+```md
+If you are new to HTML, CSS, JavaScript, etc., try our [Learn web development](/en-US/docs/Learn) tutorials.
+```
+
+在翻译时，应该将 URI 开头的 `/en-US` 替换为 `/zh-CN`：
+
+```md
+如果你刚刚开始了解 HTML、CSS、JavaScript 等相关知识，你可以试着完成我们的 [Web 开发教程](/zh-CN/docs/Learn)。
+```
+
+> 请注意，无论指向的本地化文档是否存在，都请让链接指向本地化文档。yari 平台可以在指向本地化文档的链接不存在的情况下，自动将对应的链接回落到英文版本的链接。
+
+## 中文翻译的常见问题
+
+为保证简体中文文档格式的一致性，翻译指南列出了部分规范：
```

Comment:
need to add other style doc in MDN with a brief introduction. For example, https://developer.mozilla.org/zh-CN/docs/MDN/Guidelines/Writing_style_guide
and we should indicate that when there are conflicts, following this doc.

Reply:
[This document](https://developer.mozilla.org/zh-CN/docs/MDN/Guidelines/Writing_style_guide) is mainly used for writing `en-US` documents, and it is out-of-date (some of the guidelines is used for old MDN editor, not for the new platform). We may add it after update the `Writing_style_guide`.

Reply:
Got it. Thanks!